src/app/Login/page.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+import React, { useState } from 'react';
+import ActiveButton from '@/components/Active-Button/ActiveButton';
+import InactiveButton from '@/components/Inactive-Button/InactiveButton';
+import InputField from '@/components/Input-textbox/textbox';
+import styles from './page.module.css';
+import { emailRegex } from '@/utils/constants';
+
+const isValidEmail = (email: string) => {
+  return emailRegex.test(email);
+};
+
+const LoginPage: React.FC = () => {
+  // State for email and password
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [emailError, setEmailError] = useState<string | null>(null);
```

Comment:
`setEmailError` is unused, this could be why the email error is not displaying on submit